app/views/reviews/_review.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="reviews">
+  <div class="star-rating" data-score= <%= review.rating %> ></div>
+  <p><%= review.comment %></p>
+  <% if user_signed_in? && current_user.admin == true %>
```

Comment:
kindly change the structure of this one....it does not seem right.
and use current_user.try(:admin) 
rather then going for user_signed_in? (though i am not sure it will work but give it a try)